README.md

```diff
@@ -61,6 +61,13 @@ Invoke "whoami.exe" to demonstrate running a command as another user:
   whoami.run_command
 ```
 
+Invoke "whoami.exe" with elevated privileges:
```

Comment:
Can you add a note please about the privilege(s) that would be required for this? logon as a batch job for the target user, and there are a couple for the current user, like replace a process level token and something about adjusting process memory quotas.